ocs_ci/templates/ocs-deployment/aws_gp2_csi_storage_class.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
do we need the SCs definition saved in templates? asking since they are supposed to exist automatically with OCP 4.10 from what I understand

Reply:
I wasn't sure about it, so I added them also. But I think we don't need them.